addons/l10n_ro/data/account_data.xml

```diff
@@ -21,5 +21,29 @@
         <record id="tax_group_tva_24" model="account.tax.group">
             <field name="name">TVA 24%</field>
         </record>
+        <record id="tax_group_tva_0_eu" model="account.tax.group">
```

Comment:
For the EU ones: can not we just use the existing ones?  I don't think the user is interested in seeing TVA 24% EU instead of TVA 24% after the subtotal.

Reply:
@jco-odoo I wanted so that the user to understand that this is a transaction where the VAT will be 0, not to be confused seeing tha TVA (VAT) 19% and the amount with 0, it's much more evident with EU that the tax should be 0.

Reply:
@feketemihai If it is 0, I agree we need to be clear why it is 0 in that case, but I am mainly talking about the others: for the 5% and the 19%, we could just use the tax_group_tva_24 instead of the new on.

Reply:
@jco-odoo No, the one's with previous ids was because the tax level was changed in the last 10%, first the vat rate was 24, then 20 and now it's 19%, but right now we only use 5,9, and 19, so i don't know why we should relate with previous ids, since if they will come from migration, in that group they will still have amount with 24, or 20%, if we change it and they will update the chart of account if will be more confusing, saying that the tax base is 19 EU, and the level of tax was 24%. For me it will be confusing.

Reply:
I mean: why do we have tax_group_tva_5 and tax_group_tva_5_eu.  Everything could go into tax_group_tva_5 @feketemihai

Reply:
@jco-odoo As i mentioned in the upper comment, for the 19 %, i splitted to know exactly the base, for the normal tax_group_tva_5 you also expect an amount in the VAT part, but for the one's with tax_group_tva_5_eu, the VAT amount is 0.

Reply:
@feketemihai I saw that the taxes related to tax_group_tva_5_eu have positive percentages.